bundler/src/main/chart/templates/daemonset.yaml

```diff
@@ -91,6 +91,45 @@ spec:
             {{- include "common.volumeMount" $.Values.global.volumes.input | nindent 12 }}
           resources:
             {{- toYaml $.Values.resources | nindent 12 }}
+        {{ if $.Values.filebeat.enabled -}}
+        - name: filebeat
+          image: "{{ $.Values.filebeat.imageRepo }}:{{ $.Values.filebeat.imageTag }}"
```

Comment:
it would be nice if the image had the same layout as all the other images in the repo